### ISSUE TITLE
No new data since 2022-08-05

### ISSUE THREAD

**Issue by lucasbroennimann** [metaodi/tecdottir#34]
Hi,
Unfortunately, the latest data in the csv from [stadt-zuerich](https://data.stadt-zuerich.ch/dataset/sid_wapo_wetterstationen#dataset) (and therefore also in your API) is from 2022-08-05T19:30:00+00:00 and no newer data is available. The https://www.tecson-data.ch/zurich/tiefenbrunnen/index.php is still up to date. 
I'm guessing the .csv are no longer automatically updated which is no fault of the tecdottir API here, but it would mean getting the current data would require to now access the password protected part of the tecson-data page :(

**Comment by metaodi** [comment]
Yes, I'm still trying to reinstate the access to the data or find other sources. So far lo luck 😔

**Comment by lucasbroennimann** [comment]
Thanks for the reply, sorry to hear it didn't work out so far. I used the API for a small project with my students in the last years, it was very useful to teach them about some simple API usage with python, thank you very much for providing it. I guess this semester I'll be on the lookout for alternatives.

**Comment by metaodi** [comment]
Another project of mine is [OpenERZ](https://openerz.metaodi.ch/documentation), an API that provides waste collection data for a couple of towns (i.e. on what date will cardboard be collected next).

Maybe this could be of interest to you.

**Comment by lucasbroennimann** [comment]
Thank you for the recommendation, I will have a look it it :)

**Comment by dddesign** [comment]
> Yes, I'm still trying to reinstate the access to the data or find other sources. So far lo luck 😔

any news on your side?
I wrote info@tecson.ch what/if they changed something...

**Comment by metaodi** [comment]
I know exectly what they changed: they changed the password to prevent access by any automated process because they think this causes their servers to break.

But so far they didn't want to discuss any options to adress this (less requests, more caching, generating CSV on their side, ..., etc.).

Hopefully Open Data Zurich is able to help reinstate this data source.

**Comment by metaodi** [comment]
Important disclaimer: while this is a hobby project of mine, I do work for Open Data Zurich and we're trying to get this data again. If nothing works I will go back to scrape the data from the HTML page, but I actually want to avoid doing this.

**Comment by metaodi** [comment]
Update: the data is back! I could load the missing days since 2022-08-05 and the data is updated every ~2 hours.
